### PR TITLE
[SD-5168] Formatters for NZN and category mapping based on source

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: server/aap/publish/formatters/aap_ipnews_formatter.py [hunk 1]
```diff
@@ -103,5 +116,8 @@ def format_text_content(self, tag):
         else:
             tag.replace_with('')
 
+    def _get_category_list(self, category_list):
+        return get_aap_category_list(category_list)
+
     def can_format(self, format_type, article):
         return format_type == 'AAP IPNEWS' and article[ITEM_TYPE] in [CONTENT_TYPE.TEXT, CONTENT_TYPE.PREFORMATTED]
```
Comment (mdhaman) [inline server/aap/publish/formatters/aap_ipnews_formatter.py]: Can you remove preformatted content_type?

**Comment by mdhaman** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

File: server/aap/publish/formatters/aap_ipnews_formatter.py (hunk 1 above)
Comment (marwoodandrew) [inline server/aap/publish/formatters/aap_ipnews_formatter.py]: Yep!

**Pull request merged**